### PR TITLE
Save all synchronous jobs in one container - Closes #582

### LINKED ISSUE #582: Storing synchronous actions in one place
It is currently hard to control all synchronous events that are taking place in multiple modules: blocks, transactionsPool, loader, peers. As some of them are registered after some events happening, there is also a risk of registering the same event multiple times.

I propose to store all synchronous actions in one place, so it is easy to track them and have control of how many times they were registered.

### PR DESCRIPTION
**Pull request by MaciejBaj**
- the current implementation has a bug with registering sync timer in loader module twice, which in this PR is only surrounded with try-catch.

closes #582

### CODE REVIEW

**Comment by 4miners** [comment]
- Line https://github.com/LiskHQ/lisk/blob/582-synchronous-jobs-storage/modules/loader.js#L796 is not needed, because we already register it here - https://github.com/LiskHQ/lisk/blob/582-synchronous-jobs-storage/modules/loader.js#L759
- Here is another loop - https://github.com/LiskHQ/lisk/blob/582-synchronous-jobs-storage/modules/loader.js#L83, can be also included?
- We need to break all loops if cleanup was requested - would be nice to have some global indicator for it or we can use existing `modules.blocks.isCleaning.get()`

File: helpers/jobsQueue.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var jobsQueue = {
+
+	jobsSet: {},
```
Comment (4miners) [inline helpers/jobsQueue.js]: Simple `jobs` instead of `jobsSet`

**Comment by 4miners** [comment]
Also, please note that `setInterval` and `setTimeout` behavior is not the same. `setInterval` will execute functions exactly every X ms, but `setTimeout` will execute every (actual function time + X ms), so for example in peers module:
- previous behavior: loop every 10+s (5s+ for functions + 5s `setTimeout` waiting)
- current behavior: loop every 5s

That is inconsistent with previous behavior and can lead to start new loop before previous loop ended.

**Comment by MaciejBaj** [comment]
@4miners 
**first comment**
1. yes, it is probably the reason of registering synchronous event twice
2. it is actually not a loop, just invoking function delay
3. can bo opened as separate PR
**second comment**
very good point, I will change it. After WS integration waiting timeout for response is not an issue anymore, so it would work then

**Pull request merged**